### PR TITLE
fix(client): remove wallets

### PR DESCRIPTION
**Pull request by ahmaxed**
Checklist:

- [ ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ ] My pull request targets the `main` branch of freeCodeCamp.
- [ ] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: client/src/components/Donation/DonateForm.tsx [hunk 1]
```diff
@@ -310,26 +309,13 @@ class DonateForm extends Component<DonateFormProps, DonateFormState> {
       isOneTime ? 'donate.confirm-2' : 'donate.confirm-3',
       { usd: donationAmount / 100 }
     )}:`;
-
-    const walletlabel = `${t(
-      isOneTime ? 'donate.wallet-label' : 'donate.wallet-label-1',
```
Comment (ojeytonwilliams) [inline client/src/components/Donation/DonateForm.tsx:315]: If this is going, should the translation jsons be updated?

Comment (moT01) [reply client/src/components/Donation/DonateForm.tsx:315]: I don't think that will cause any problems - they will just be unused keys, right? Which should be fine.

Comment (ahmadabdolsaheb) [reply client/src/components/Donation/DonateForm.tsx:315]: no cause it will be coming back? would it be a problem if there is an unused key in the json?

**Review by moT01 — APPROVED** [review]
LGTM 🎉 

No errors or anything - opened up all three donation forms

File: client/src/components/Donation/DonateForm.tsx (hunk 1 above)
Comment (ojeytonwilliams) [reply client/src/components/Donation/DonateForm.tsx:315]: It's not a problem except that it's more clutter and another thing that people think they should translate. I don't think it'll break anything, though.

Comment (ojeytonwilliams) [reply client/src/components/Donation/DonateForm.tsx:315]: If it's going to be used, it's totally fine to leave it.

**Pull request merged**